docker/docker-compose.dev.yml

```diff
@@ -58,7 +67,7 @@ services:
         command: ''
         entrypoint: ['/bin/sh', '-c', 'sleep infinity']
     db-dev:
-        image: postgres:15.4
+        image: pgvector/pgvector:pg16
```

Comment:
could this be a problem with people running on selfhosting, do we need to update our helm ?

Reply:
We probably need to also update `docker-compose.yml` then 

> postgres:15.4

Reply:
this change seems dangerous for anyone running the docker image for prod on self-hosting

Reply:
rollbacked